python/tvm/relay/frontend/tensorflow.py

```diff
@@ -788,6 +788,14 @@ def _impl(inputs, attr, params, mod):
     return _impl
 
 
+def _expm1():
+    def _impl(inputs, attr, params, mod):
+        lh = get_relay_op("exp")(inputs[0])
+        return get_relay_op("subtract")(lh, tvm.relay.const(1, attr["T"].name))
```

Comment:
Not entirely sure, but can't you write `return lh -1`? Isn't there an overload for the `-` operator?

Reply:
The operator` - `can be used, but cannot be written as` lh - 1`. I replaced "subtract" with `exp_out-1.`